src/ray/rpc/worker/core_worker_client.h

```diff
@@ -312,7 +312,8 @@ class CoreWorkerClient : public std::enable_shared_from_this<CoreWorkerClient>,
         callback(status, reply);
       };
 
-      INVOKE_RPC_CALL(CoreWorkerService, PushTask, *request, rpc_callback, grpc_client_);
+      (void)INVOKE_RPC_CALL(CoreWorkerService, PushTask, *request, rpc_callback,
```

Comment:
use `RAY_UNUSED`

Reply:
Oh ok thanks, didn't realize that exists.